ddtrace/__init__.py

```diff
@@ -24,11 +24,11 @@
 ]
 
 
-@deprecated("This method will be removed altogether", "1.0.0")
+@remove(removal_version="1.0.0")
```

Comment:
removed annotation will log the following warning: `Using function/method install_excepthook is deprecated and will be removed in version '1.0.0'`

Reply:
I fixed my comment